portfolio/src/main/java/com/google/sps/servlets/SubmitEmailServlet.java

```diff
@@ -19,12 +26,26 @@
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
     // Get the value entered in the form.
-    String email = request.getParameter("email-address");
+    // String email = request.getParameter("email-address");
+
+    String email = Jsoup.clean(request.getParameter("email-address"), Whitelist.none());
+    long timestamp = System.currentTimeMillis();
+
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    KeyFactory keyFactory = datastore.newKeyFactory().setKind("Email");
+    FullEntity taskEntity =
+        Entity.newBuilder(keyFactory.newKey())
+            .set("title", email)
+            .set("timestamp", timestamp)
+            .build();
+    datastore.put(taskEntity);
+
+    response.sendRedirect("/index.html");
 
     // Print the value so you can see it in the server logs.
     System.out.println("User's email: " + email);
 
     // Write the value to the response so the user can see it.
-    response.getWriter().println("Thank you for your response! I will be reaching out to you soon");
+    //response.getWriter().println("Thank you for your response! I will be reaching out to you soon");
```

Comment:
If not needed, can you remove this line?

Reply:
Yes, removed this line